gatsby-config.ts

```diff
@@ -2,36 +2,30 @@ import type { GatsbyConfig } from "gatsby";
 
 const config: GatsbyConfig = {
   siteMetadata: {
-    title: `UndegroundJS`,
-    siteUrl: `https://undergroundjs.com`
+    title: `UndergroundJS`,
+    description: `Nashville's ultimate JavaScript event on September 21, 2023! The single-track conference focuses on JavaScript & the Nashville dev community.`,
```

Comment:
Can we use the text jerome posted in slack?

Reply:
This is the one liner from here: https://techfednash.slack.com/archives/GFYCA9CP7/p1681501706469489 I just expanded JS to JavaScript and Nash to Nashville. Metadata descriptions have a character limit ~155 on Google so we can't use the full one.